scripts/install

```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-export SCRIPT_DIR="$(cd "$(dirname "$0")/.." && pwd)"
+source "$(dirname -- "$0")/vars"
```

Comment:
I'm worried how `dirname -- "$0"` will play with the binary created from the homebrew formula, which is a symlink. Could you please test that locally before merging the PR?